calcrule_unconditional_cash_payment/calculation_rule.py

```diff
@@ -0,0 +1,128 @@
+    signal_get_linked_class = Signal(providing_args=[])
+    signal_calculate_event = Signal(providing_args=[])
+    signal_convert_from_to = Signal(providing_args=[])
+
+    @classmethod
+    def ready(cls):
+        now = datetime.datetime.now()
+        condition_is_valid = (now >= cls.date_valid_from and now <= cls.date_valid_to) \
+            if cls.date_valid_to else (now >= cls.date_valid_from and cls.date_valid_to is None)
+        if condition_is_valid:
+            if cls.status == "active":
+                # register signals getParameter to getParameter signal and getLinkedClass ot getLinkedClass signal
+                cls.signal_get_rule_name.connect(cls.get_rule_name, dispatch_uid="on_get_rule_name_signal")
+                cls.signal_get_rule_details.connect(cls.get_rule_details, dispatch_uid="on_get_rule_details_signal")
+                cls.signal_get_param.connect(cls.get_parameters, dispatch_uid="on_get_param_signal")
+                cls.signal_get_linked_class.connect(cls.get_linked_class, dispatch_uid="on_get_linked_class_signal")
+                cls.signal_calculate_event.connect(cls.run_calculation_rules, dispatch_uid="on_calculate_event_signal")
+                cls.signal_convert_from_to.connect(cls.run_convert, dispatch_uid="on_convert_from_to")
+
+    @classmethod
+    def active_for_object(cls, instance, context, type="account_payable", sub_type="cash_payment"):
+        return instance.__class__.__name__ == "Policy" \
+               and context in ["policyCreated"] \
+               and cls.check_calculation(instance)
+
+    @classmethod
+    def check_calculation(cls, instance):
+        class_name = instance.__class__.__name__
+        match = False
+        if class_name == "ABCMeta":
+            match = str(cls.uuid) == str(instance.uuid)
+        elif class_name == "PaymentPlan":
+            match = str(cls.uuid) == str(instance.calculation)
+        elif class_name == "Policy":
+            match = cls.check_calculation(instance.product)
+        elif class_name == "Product":
+            # if product → paymentPlans
+            payment_plans = PaymentPlan.objects.filter(benefit_plan=instance, is_deleted=False)
+            for pp in payment_plans:
+                if cls.check_calculation(pp):
+                    match = True
+                    break
+        return match
+
+    @classmethod
+    def calculate(cls, instance, **kwargs):
+        context = kwargs.get('context', None)
+        user = kwargs.get('user', None)
+        class_name = instance.__class__.__name__
+        if class_name == "Policy":
+            product = instance.product
+            payment_plan = PaymentPlan.objects.filter(benefit_plan=product, calculation=cls.uuid, is_deleted=False)
+            if payment_plan:
+                payment_plan = payment_plan.first()
+                cls.run_convert(instance=instance, context=context, convert_to='Bill', payment_plan=payment_plan,
```

Comment:
@wzglinieckisoldevelo Please write this in such way as below:
`run_convert(
    instance=instance, 
   context=context, 
   convert_to='Bill', 
   payment_plan=payment_plan,
)
`